src/App.js

```diff
@@ -1,11 +1,24 @@
 import React from "react";
+import { Route } from 'react-router-dom';
 import Header from "./components/Header.js";
 
 
+
+import WelcomePage from './components/WelcomePage';
+import CharacterList from './components/CharacterList';
+import SearchForm from './components/SearchForm';
+
 export default function App() {
   return (
     <main>
-      <Header />
-    </main>
-  );
+    <Header />
+        <div>
+            <Route exact path="/" component={WelcomePage} />
+            <Route path="/characters" component={CharacterList} />
+            <Route path="/search" component={SearchForm} />
+        </div>
+        </main>
+    )
 }
```

Comment:
check the indentation! Using a formatter will help with this, but make sure that you indent every time you "go down a level", and then reverse-indent when you move back up a level. This code would look like:
```
export default function App() {
  return (

    <main>

      <Header />

      <div> // This div isn't inside the header, so it's on the same indentation level

        <Route exact path="/" component={WelcomePage} />

        <Route path="/characters" component={CharacterList} />

        <Route path="/search" component={SearchForm} />

      </div>

    </main>
  )
}
```